report/templates/sample.html

```diff
@@ -49,4 +49,9 @@ <h2>Run logs</h2>
 <pre>
 {{ run_logs }}
 </pre>
+
+<h2>Triage</h2>
+<pre>
+{{ triage }}
+</pre>
```

Comment:
Would it be a good idea to move this to the top of the file?
I reckon this is a pretty important conclusion about the fuzz target to state at the beginning, what do you think?

Reply:
BTW, shall we also add a new column at [benchmark.html](https://github.com/google/oss-fuzz-gen/blob/main/report/templates/benchmark.html#L33) to show if LLM thinks this crash is caused by the fuzz target or the project?